scripts/add_ts_data_bidmatrixinfo.py

```diff
@@ -0,0 +1,59 @@
+###this script adds time series data to the bid matrix information objects in the powerops client
+
+from cognite.client.data_classes import TimeSeries
+from cognite.powerops import PowerOpsClient
+import random
+import datetime
+
+
+power = PowerOpsClient.from_settings()
+cdf = power.cdf
+
+start_time = int((datetime.datetime.now() - datetime.timedelta(days=365)).timestamp() * 1000)
+end_time = int(datetime.datetime.now().timestamp() * 1000)
+
+# creating time series, with data points every hour
+if not cdf.time_series.retrieve(external_id="wim_ts_hourly"):
+    wim_ts_hourly = cdf.time_series.create(TimeSeries(name="wim_ts_hourly", external_id="wim_ts_hourly"))
+    data_points = [
+        {"timestamp": start_time + i * 3600 * 1000, "value": random.random()}
+        for i in range(int((end_time - start_time) / (3600 * 1000)))
+    ]
+    cdf.time_series.data.insert(datapoints=data_points, external_id="wim_ts_hourly")
+
+# creating an empty time series, without a name to check the external_id default
+if not cdf.time_series.retrieve(external_id="wim_ts_empty"):
+    wim_ts_empty = cdf.time_series.create(TimeSeries(external_id="wim_ts_empty"))
+
+tsList = ["wim_ts_hourly", "wim_ts_empty"]
+
+
+def get_random_timeseries():
+    choice = random.choice([1, 2, 3])
+    if choice == 1:
+        return [random.choice(tsList)]  # Velg en tidsserie
+    elif choice == 2:
+        return random.sample(tsList, 2)  # Velg to tidsserier
+    else:
+        return tsList
+
+
+# Adding the timeseries to the list of matrices to upsert
+upsertlist = []
+
+for matrix in power.v1.day_ahead_bid.bid_matrix_information.list():
+    matrix.alerts = None
+    exid = matrix.external_id
+    mat = matrix.as_write()
+    mat.external_id = exid
```

Comment:
**note:** best to add a comment here about _why_ we're doing this (ie explain it's a workaround because there's a bug in the `.as_write()` that was overwriting the external_id so future users know that it can be removed once the bug is fixed